### ISSUE TITLE
Remove app patch feature

### ISSUE THREAD

**Issue by mpost** [eclipsesource/tabris-js#1672]
The experimental app patch feature is not used widely and with the recent shortened app approval process in the apple app store is not a necessity anymore.

Relates to https://github.com/eclipsesource/tabris-js/issues/553 and https://github.com/eclipsesource/tabris-js/issues/509

**Comment by sand123** [comment]
Hi, I constantly use installPatch feature, because we do not publish app in playmarket - it's a small app for internal tasks for 10+ users. Would it better to mark it in documentation to use with caution? Otherwise it would be much harder for me to push updates. Thanks

**Comment by cookieguru** [comment]
@sand123 If it's a small group of trusted users you could consider one of the dozens of beta app distribution mechanisms, many of which have the ability to auto update (or at least notify).

**Comment by mpost** [comment]
I would agree. With the tester channels on the play store and in testflight you get much better insights into your apps usage and also nearly instant roll-out. With the patching features there are certain scenarios where we can not guarantee a patch is applied to its fullest, making it hard to recommend it as a general solution.
